Sources/Filters/General/ImageCropFilter/index.d.ts

```diff
@@ -1,10 +1,11 @@
 import { vtkAlgorithm, vtkObject } from "../../../interfaces";
+import { Extent } from "../../../types";
 
 /**
  *
  */
-interface IImageCropFilterInitialValues {
-	croppingPlanes?: any;
+export interface IImageCropFilterInitialValues {
+	croppingPlanes?: Extent;
```

Comment:
is it really `Extent`? I haven't checked but that seems strange...

Reply:
No but it's an array of 6 elements, i wanted to make a local type CroppingPlanes, should i do it?

Reply:
I think it use to be 6 vtkPlane but now it can be as many as we want...

Reply:
from the code it's 6 number not vtkPlane

Reply:
You right, @floryst mentioned to me that the ImageCropFilter was indeed using some ijk bounds. Therefore using extent make sense. It might be off by 1 in the true meaning of extent but that is fine for the type definition...